fluss-server/src/test/java/com/alibaba/fluss/server/testutils/FlussClusterExtension.java

```diff
@@ -115,7 +115,7 @@ public final class FlussClusterExtension
     private File tempDir;
 
     private final Map<Integer, TabletServer> tabletServers;
-    private final List<ServerInfo> tabletServerInfos;
+    private final Map<Integer, ServerInfo> tabletServerInfos;
```

Comment:
Use `List<ServerInfo> tabletServerInfos` will cause two `ServerInfo` with same tablet server id which will then cause `newTabletServerClientForNode` retrun a client connecting to old ServerNode.